### ISSUE TITLE
As a user I want to log in

### ISSUE THREAD

**Issue by lcabrini** [AluguntuguiProject/aluguntugui#3]

**Comment by lcabrini** [comment]
I guess Auth is the correct module for this, am I right?

**Comment by lcabrini** [comment]
I propose this to be added to the sprint backlog for sprint #1

**Comment by benbaka** [comment]
Log in and Log out functionality would fit into sprint (or milestone 1). 

When the user logs in he sees a blank page with a log out button to the top right hand corner of the screen. 

Once the above is completed, the story is considered complete.

Your thoughts ?

**Comment by lcabrini** [comment]
Yes, I agree. I think the empty page will be a placeholder for what will later become the user's dashboard. So can I move this to sprint backlog?

**Comment by benbaka** [comment]
Yep.

**Comment by benbaka** [comment]
@lcabrini 
For now it is possible for registered users to authenticate and log in. 

But their sessions are not remembered and if they were ever to land on the login page they wouldn't be redirected. 

Should I create a user story as backlog so that we can continue with the user stories in this current sprint or implement that functionality right here and now ?

**Comment by benbaka** [comment]
@lcabrini 
You can check out my latest additions to code. I am making use of seed to get me a user for testing the flow.

**Comment by lcabrini** [comment]
Yes, I had forgot to commit my own seed, so it got conflicted. But, I can see that your solution was better, since it starts by checking if admin already exists.
